ChangeLog

```diff
@@ -33,7 +37,7 @@
     * Fixed PUT upload when folders within the path does not exist
     * Cameras! Support for:
         - V4L2 cameras (webcams - MJPEG and YUYV formats supported)
-        - picamera2 (libcamera stack) (slow)
+        - picamera2 (libcamera stack) (slow) (removed)
```

Comment:
No, please, add *removed picamera2* to new changelog - don't update history of published version